### PR TITLE
querySelector behavior fixed (closes #922)

### LINKED ISSUE #922: [todomvc] Js-error appears on the on the VanillaJS demo page
pages: http://todomvc.com/examples/vanillajs/ http://todomvc.com/examples/vanilla-es6
error: `Uncaught TypeError: Cannot set property 'className' of null`

reproduced in playground. Tested in Chrome but should be checked in other browsers

### PR DESCRIPTION
**Pull request by georgiy-abbasov**
\cc @churkin, @AlexanderMoskovkin, @VasilyStrelyaev

### CODE REVIEW

File: test/client/fixtures/sandbox/node/document-test.js [hunk 1]
```diff
@@ -547,3 +547,16 @@ test('an iframe should not contain self-removing scripts after document.close (G
 
     strictEqual(nativeMethods.querySelectorAll.call(document, '.' + SHADOW_UI_CLASSNAME.selfRemovingScript).length, 0);
 });
+
+test('querySelector should return element if selector contains href attribute with hash as value (GH-922)', function () {
```
Comment (AlexanderMoskovkin) [inline test/client/fixtures/sandbox/node/document-test.js]: `querySelector should return an element if a selector contains the href attribute with hash as a  value (GH-922)`

by @MarinaRukavitsyna

**Comment by georgiy-abbasov** [comment]
ping @churkin 
\cc @LavrovArtem

**Comment by AlexanderMoskovkin** [comment]
@testcafe-build-bot \retest

**Comment by georgiy-abbasov** [comment]
@testcafe-build-bot \retest

**Pull request merged**